src/WebUI/Resources.js

```diff
@@ -95,4 +95,5 @@ define("Environments/Providers/Kubernetes/Resources", ["require", "exports"], fu
     exports.LoadingPodsSpinnerLabel = "Loading associated pods...";
     exports.LoadingServicesSpinnerLabel = "Loading services...";
     exports.CopyExternalIp = "Copy External IP to clipboard";
+    exports.ImageDetailsUnavailableText = "Image details only available for images built in Azure Pipelines";

```

Comment:
Image details available only for images built in Azure Pipelines

only placement may be changed, talk to someone who are good at english

Reply:
Updated the text